<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>wdgBiometrics</class>
 <widget class="QWidget" name="wdgBiometrics">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1012</width>
    <height>669</height>
   </rect>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_3">
     <item>
      <widget class="QLabel" name="lbl">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Your biometric information</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="wdgYearMonth" name="wdgYM" native="true"/>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="myQTableWidget" name="tblBiometrics">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderStretchLastSection">
          <bool>false</bool>
         </attribute>
         <attribute name="verticalHeaderVisible">
          <bool>false</bool>
         </attribute>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="layChart"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QLabel" name="lblFound">
       <property name="text">
        <string>Registers found</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <action name="actionBiometricsNew">
   <property name="icon">
    <iconset resource="../images/caloriestracker.qrc">
     <normaloff>:/caloriestracker/add.png</normaloff>:/caloriestracker/add.png</iconset>
   </property>
   <property name="text">
    <string>New biometric information</string>
   </property>
   <property name="toolTip">
    <string>New biometric information</string>
   </property>
  </action>
  <action name="actionBiometricsDelete">
   <property name="icon">
    <iconset resource="../images/caloriestracker.qrc">
     <normaloff>:/caloriestracker/list-remove.png</normaloff>:/caloriestracker/list-remove.png</iconset>
   </property>
   <property name="text">
    <string>Delete biometric information</string>
   </property>
   <property name="toolTip">
    <string>Delete biometric information</string>
   </property>
  </action>
  <action name="actionBiometricsEdit">
   <property name="icon">
    <iconset resource="../images/caloriestracker.qrc">
     <normaloff>:/caloriestracker/document-edit.png</normaloff>:/caloriestracker/document-edit.png</iconset>
   </property>
   <property name="text">
    <string>Edit biometric information</string>
   </property>
   <property name="toolTip">
    <string>Edit biometric information</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>myQTableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">myqtablewidget</header>
  </customwidget>
  <customwidget>
   <class>wdgYearMonth</class>
   <extends>QWidget</extends>
   <header location="global">wdgYearMonth</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>tblBiometrics</tabstop>
 </tabstops>
 <resources>
  <include location="../images/caloriestracker.qrc"/>
 </resources>
 <connections/>
</ui>
